src/System.Memory/src/System.Memory.csproj

```diff
@@ -100,20 +100,20 @@
     <Compile Include="..\Common\src\System\MutableDecimal.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true'">
+    <Compile Include="$(CommonPath)\CoreLib\System\Memory.cs" />
```

Comment:
should we introduce a variable for the shared folder? e.g. `$(SharedPath)` or `$(SharedCommonPath)`

Reply:
I am not sure if it is worth it.

Reply:
Replacing all existing paths starting with $(CommonPath)\CoreLib works if we ever change the place of the mirror. So I'm fine with hardcoding it.

Reply:
All these path variables just makes it harder for people to find the files. It's not like msbuild supports "hitting F12 to go to the definition of path variable." 

Not a good tradeoff for the remote chance that the mirror location will change, especially since search-replace will work just fine in that case.

Reply:
I agree. It's good that we talked about it.